### PR TITLE
Add performance testing to conformance tests

### PR DESCRIPTION
**Pull request by stxue1**
- Added some integration tests under `integration/`
- Added `performance_testing.py` to run and time tests and output results to a CSV file
- Added `create_graph.py` to read from CSV file (or run performance testing itself) and display/output graphs
- Added support for runner specific arguments in `run.py`
- Tests will run on individual processes instead of threads in order to properly time the tests
- Refactored `run.py` in order to be callable + separated out common functions into `lib.py`
- Added `merge.py` to merge CSV files
- `dir` field in `conformance.yaml` is now specified from project root directory
- Added `id` field to `conformance.yaml`
- Added `--id`, `--repeat`, `--exclude-numbers` for better specification when running tests

### CODE REVIEW

File: performance_testing.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                                           help='Specify the output CSV file.')
+    performance_testing_group.add_argument("--all-runners", "-a", default=False, action="store_true",
+                                           help="Specify whether to run with all runners. This will override the "
+                                                "--runners argument.")
+    performance_testing_group.add_argument("--runners", default=None,
+                                           help="Specify multiple runners in a comma separated list.")
+
+
+def main(args):
+    parser = argparse.ArgumentParser()
```
Comment (adamnovak) [inline performance_testing.py:103]: This should probably have some top-level help text to explain what the tool is for or how to use it.

File: performance_testing.py
Comment (adamnovak) [inline performance_testing.py:1]: This should probably be `run_performance.py` or something else that sounds more like a command and not a library.

File: create_graph.py [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+                                  "fileextension]. For example, if wdl_graph.png is passed, then the first file created"
+                                  " will be wdl_graph1.png.")
+    # "default" is a marker as the default will be generated at runtime
+    output_args.add_argument("--dimensions", const="default", default=None, nargs="?", action="store",
+                             help="If custom dimensions are needed, this can be called with input format x_size,y_size "
+                                  "in inches. Calling this with no value will size the graph accordingly.")
+
+
+def main(args):
+    parser = argparse.ArgumentParser()
```
Comment (adamnovak) [inline create_graph.py:345]: This script could also use some top-level help.

**Review by adamnovak — CHANGES_REQUESTED** [review]

File: integration/tut01/helloHaplotypeCaller.wdl
[the remark this answers is not in the document]
Comment (stxue1) [reply integration/tut01/helloHaplotypeCaller.wdl:1]: I modified some of the original files from the [WDL GATK tutorials](https://support.terra.bio/hc/en-us/articles/360037127992--1-howto-Write-your-first-WDL-script-running-GATK-HaplotypeCaller), which links to a [github repo](https://github.com/openwdl/wdl-docs) that has a BSD3 license. I probably could move them into S3, but I did modify the files so I'm not sure how pulling files in from S3 would work then.

**Pull request merged**